sqlmesh/core/dialect.py

```diff
@@ -98,32 +98,6 @@ class StagedFilePath(exp.Table):
     """Represents paths to "staged files" in Snowflake."""
 
 
-def _scan_var(self: Tokenizer) -> None:
```

Comment:
so we basically got rid of this and now it's handled at parse time?

Reply:
yea